app-web/src/components/GithubTemplate/Navigation/Navigation.js

```diff
@@ -73,6 +73,16 @@ const Navigation = ({ items }) => {
     );
   }
 
+  if (groupedResources[RESOURCE_TYPES.EVENTS]) {
```

Comment:
instead of pushing to the groups array in the if statement separately, why not just map it?